src/pages/Card/_components/CardBox.tsx

```diff
@@ -1,41 +1,39 @@
-import { useState } from 'react';
 import styled from 'styled-components';
 
 import CardContent from '@/pages/Card/_components/CardContent';
-import { CARD_DATA } from '@/pages/Card/_constants/cardData';
 
 interface CardBoxProps {
-  cardBoxTitle: string;
+  tag: string;
+  cards: {
+    id: number;
+    cardTitle: string;
+    cardSrc: string;
+    cardTarget: string;
+    cardInfo: string;
+    hasEvent: boolean;
+    tags: string[];
+  }[];
+  isLast: boolean;
 }
```

Comment:
인터페이스를 잘 선언해주셨는데, card 타입을 따로 빼면 더 확장성 높은 코드가 될 것 같아유
그리고 여기서 `interface Card `을 `_interfaces` 폴더로 따로 뺀 후 `typeof CARD_DATA)[0]['card'];` 대신 사용해주셔도 가독성 좋고 재사용성 좋은 코드가 될 것 같아유

```suggestion
interface Card {
  id: number;
  cardTitle: string;
  cardSrc: string;
  cardTarget: string;
  cardInfo: string;
  hasEvent: boolean;
  tags: string[];
}

// Generic 사용
interface CardBoxProps<T = Card> {
  cardBoxTitle: string;
  tag: string;
  cards: T[]; 
  isLast: boolean;
}

// or

interface CardBoxProps {
  cardBoxTitle: string;
  tag: string;
  cards: Card[]; 
  isLast: boolean;
}

```

Reply:
오오 interface 폴더 따로 분리하면 더 효율적인 코드가 되겠네요! 감사합니다!